packages/puppeteer-core/src/common/WebWorker.ts

```diff
@@ -84,18 +91,19 @@ export class WebWorker extends EventEmitter {
     super();
     this.#client = client;
     this.#url = url;
+    this.#world = new IsolatedWorld(this, new TimeoutSettings());
```

Comment:
do workers have DOM? can we create isolated worlds there?

Reply:
They don't have DOM, but we only use the execution related portion of isolated world here.

Reply:
hm, maybe we should keep using the execution context for better readability? I would say it should be `realm` instead of the world here and it's realm().sandbox() should say `main` or `default`. And when we have a secondary world in frames that should be a realm too but with the different sandbox name.

Reply:
Discussed offline.